ecl/agentexec/agentexec.cpp

```diff
@@ -386,8 +390,11 @@ class WaitThread : public CInterfaceOf<IPooledThread>
     StringAttr queue;
     Linked<IJobQueueItem> item;
     bool isThorAgent = false;
+    static unsigned currentGraphNumber;
```

Comment:
I don't think the setJobName modulation is the right approach.
Instead, I suggest an atomic, which WaitThread increments at construction and decrements at destruction, and stores the current value as it's instance number:
e.g.:
```
static std::atomic<unisnged> activeWaitThreads{0};

WaitThread()
{
    myInstanceNumber = ++activeWaitThreads; // equiv to activeWaitThreads.fetch_add(1) + 1
...
}

~WaitThread()
{
    --activeWaitThreads;
}